src/Bot/Log/ListTable.php

```
@@ -356,73 +359,73 @@ protected function handle_row_actions($item, $column_name, $primary) {
-					<option value=""><?php echo __('All Source Types', lineconnect::PLUGIN_NAME); ?></option>
-					<?php echo lineconnect::makeHtmlSelectOptions(\Shipweb\LineConnect\Bot\Constants::WH_SOURCE_TYPE, $_REQUEST['source_type'] ?? null); ?>
+					<option value=""><?php echo __( 'All Source Types', lineconnect::PLUGIN_NAME ); ?></option>
+					<?php echo lineconnect::makeHtmlSelectOptions( \Shipweb\LineConnect\Bot\Constants::WH_SOURCE_TYPE, $_REQUEST['source_type'] ?? null ); ?>
 				</select>
 				<select name="bot_id" id="bot_id">
-					<option value=""><?php echo __('All Channels', lineconnect::PLUGIN_NAME); ?></option>
+					<option value=""><?php echo __( 'All Channels', lineconnect::PLUGIN_NAME ); ?></option>
 					<?php
-					foreach (lineconnect::get_all_channels() as $key => $value) {
-						echo '<option value="' . $value['prefix'] . '" ' . (isset($_REQUEST['bot_id']) && $value['prefix'] === $_REQUEST['bot_id'] ? 'selected="selected"' : '') . '>' . $value['name'] . '</option>';
+					foreach ( lineconnect::get_all_channels() as $key => $value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)